ContentBrowser/src/main/java/com/amazon/android/contentbrowser/ContentBrowser.java

```diff
@@ -2779,6 +2779,28 @@ private ContentContainer createFavoritesContentContainer(ContentContainer parent
         return rootFavorites;
     }
 
+    public ContentContainer getHeroSliderContentContainer() {
+        ContentContainer root = getRootContentContainer();
+        ContentContainer contentContainer = root.findContentContainerByName(ZypeSettings.ROOT_SLIDERS_PLAYLIST_ID);
+
+        if (contentContainer == null) {
+            ContentContainer sliderContainer = createHeroSliderContentContainer(root);
+            root.getContentContainers().add(sliderContainer);
+            contentContainer = sliderContainer;
+        }
+
+        return contentContainer;
+    }
+
+    private ContentContainer createHeroSliderContentContainer(ContentContainer parent) {
+        ContentContainer rootFavorites = new ContentContainer(ZypeSettings.ROOT_SLIDERS_PLAYLIST_ID);
```

Comment:
Can we rename to `rootSliders` to not be confused?

Reply:
addressed, thanks

Reply:
oh I'm sorry, I mean to rename the `rootFavorites` to `rootSliders` in the row
```
ContentContainer rootFavorites = new ContentContainer(ZypeSettings.ROOT_SLIDERS_PLAYLIST_ID);
```

Reply:
fixed

Reply:
Looks like you changed the `rootFavorites` in `createFavoritesContentContainer()`. Is it intended?
Please check your commit https://github.com/zype/zype-firebuilder/pull/21/commits/d59cfed77c4cfe45a7ade0bf86d7bb7de7d232c8. You added `createHeroSliderContentContainer()` and there was `rootFavorites` variable in it. I only mean to change that variable to make its name consistent.

Reply:
addressed, pls check it thanks